tests/registration/test_registration.py

```diff
@@ -13,35 +15,136 @@
     name="(!) WAIT LINK. Description of the tested functionality",
 )
 class TestRegistration:
-    @title("User registration under minimum requirements")
-    @description(
-        "WHEN the user submits the minimum required data for registration, "
-        "THEN the information about the user is stored in the database"
-    )
-    def test_of_registration(self, postgres):
-        with step("Generation data for registration"):
-            email = generate_string(length=2, additional_characters=["@te.st"])
+    """
+    Test suite for testing the registration of a new user
```

Comment:
It is not necessary to add a description to the tests. Title and description should explain everything

Reply:
Fixed